dom.bs

```diff
@@ -613,7 +614,7 @@ steps:
    <li><p>If <var>path</var>[<var>index</var>]'s <a for=Event/path>root-of-closed-tree</a> is true,
    then increase <var>currentTargetHiddenSubtreeLevel</var> by 1.
 
-   <li><p>If <var>path</var>[<var>index</var>]'s <a for=Event/path>item</a> is
+   <li><p>If <var>path</var>[<var>index</var>]'s <a for=Event/path>current target</a> is
```

Comment:
*current target* being different form *currentTarget* is super confusing here...
Perhaps we should call *current target* in the struct *invocation target* instead??